qmqtt.pri

```diff
@@ -0,0 +1,42 @@
+INCLUDEPATH += $$PWD/src/mqtt
```

Comment:
Can you add a comment here to explain the purpose of this file?
There are many build file already in this project, and this one is a bit special